huaweicloud/services/acceptance/cdn/resource_huaweicloud_cdn_domain_test.go

```diff
@@ -582,8 +613,8 @@ func TestAccCdnDomain_configs(t *testing.T) {
 				ImportStateVerify: true,
 				ImportStateIdFunc: testCDNDomainImportState(resourceName),
 				ImportStateVerifyIgnore: []string{
-					"enterprise_project_id", "configs.0.url_signing.0.key", "configs.0.https_settings.0.certificate_body",
-					"configs.0.https_settings.0.private_key", "cache_settings",
+					"enterprise_project_id", "configs.0.url_signing.0.key", "configs.0.url_signing.0.backup_key",
+					"configs.0.https_settings.0.certificate_body", "configs.0.https_settings.0.private_key", "cache_settings",
```

Comment:
Please add `configs.0.url_signing.0.backup_key` in document  `import ignore` part.